<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseTuple</class>
 <widget class="QWidget" name="CaseTuple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedicalIntelligence (Doctor Client)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="m_diagnosis"/>
   </item>
   <item>
    <widget class="QLineEdit" name="m_prescription"/>
   </item>
   <item>
    <widget class="QLineEdit" name="m_advice"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
